cmake/modules/SwiftSource.cmake

```diff
@@ -252,7 +319,7 @@ function(_compile_swift_files
   # FIXME: Cleaner way to do this?
   if(SWIFTFILE_IS_STDLIB_CORE)
     list(APPEND swift_flags
-        "-nostdimport" "-parse-stdlib" "-module-name" "Swift")
+        "-nostdimport" "-parse-stdlib")
```

Comment:
Are we able to drop -module-name "Swift" here. I thought they were orthogonal things... i.e. otherwise, we wouldn't be able to compile things like simd outside of stdlibCore. Or am I missing something?

Reply:
This dates back from when the module name was inferred from the first file in a Swift library target. It used to be necessary but now it's specified explicitly everywhere.